common/djangoapps/student/models.py

```diff
@@ -1002,7 +1005,6 @@ def __unicode__(self):
         ).format(self.user, self.course_id, self.created, self.is_active)
 
     @classmethod
-    @transaction.atomic
```

Comment:
By looking at how the API modifies data (e.g., how post requests are handled). We don't want to disable atomicity if the API creates records that depend on each other in a way that would leave the system in a bad state if a query were to fail.